### ISSUE TITLE
fix pypy support?

### ISSUE THREAD

**Issue by ThomasWaldmann** [dcwatson/deflate#35]
after adding testing to the automated wheel building running via github actions, i noticed that the pypy wheel does not test ok. likely no new issue, just nobody noticed yet / complained yet.

when this issue is fixed, the pypy wheel building can be re-enabled.

see there:
- c2ebdba78f45531d64c6b9158d5e6bce66b93c01
- 388796916273fe60b455d3eb6316c55c0492212e
- #36 normal github actions CI with pypy-3.8 also fails (so it is a general issue, not just with the wheel)

**Comment by ThomasWaldmann** [comment]
just noted that the deflate 0.3.0 pypi release did not have a pypy wheel.

**Comment by ThomasWaldmann** [comment]
Depending on what the goal will be with supporting PyPy or not, we could adjust the pypi metadata in setup.py accordingly:
```
        'Programming Language :: Python :: Implementation :: CPython',
        'Programming Language :: Python :: Implementation :: PyPy'
```

**Comment by dcwatson** [comment]
Seems like it's blocked on this:

https://foss.heptapod.net/pypy/pypy/-/issues/3837

I don't know if this is something we could work around or not, but not having used PyPy myself, I'm not inclined to spend much effort on it.

**Comment by dcwatson** [comment]
Seems like this would probably require using `ctypes` or `CFFI` - I'd imagine that would be a separate wrapper library. Closing this out.

**Comment by henryiii** [comment]
Fixed in #50.
